examples/player/simid_player.html

```diff
@@ -35,11 +35,39 @@
       simidPlayer.initializeAd();
     }
 
+    function toggleButtons() {
+      let adRequests = document.querySelectorAll(".ad_request");
+      adRequests.forEach((request) => {
+        if (request.disabled == false) {
+          request.disabled = true;
+        }
+        else {
+          request.disabled = false;
+        }
+      });
+    }
+
     function playAd() {
       if (!simidPlayer) {
         initAd();
       }
       simidPlayer.playAd();
+      toggleButtons();
+    }
+
+    function closeAd() {
+      simidPlayer.playerCloseAd();
```

Comment:
imo these should still have a guard `if (!simidPlayer) {return}`
I know these buttons should only show up after simidPlayer is defined, but its good to code defensively whenever possible

https://en.wikipedia.org/wiki/Defensive_programming#:~:text=Defensive%20programming%20is%20a%20form,of%20software%20under%20unforeseen%20circumstances.&text=Defensive%20programming%20is%20an%20approach,of%20software%20bugs%20and%20problems.

Reply:
okay, so I will add the guard as well